llm_api_hub/server/app.py

```
@@ -310,7 +320,7 @@ def validate_auth(request: Request) -> None:
 
 
 def configured_models() -> List[str]:
-    models = ["wrapper/deepseek", "wrapper/mistral", "wrapper/claude", "wrapper/gemini", "wrapper/colab"]
+    models = ["wrapper/deepseek", "wrapper/mistral", "wrapper/grok", "wrapper/claude", "wrapper/gemini", "wrapper/colab"]
```

Pylint (4.0.6)

[convention] Line too long (121/100)

(C0301)